icechunk-python/tests/test_zarr/test_store/test_icechunk_store.py

```diff
@@ -301,6 +303,12 @@ async def test_set_if_not_exists(self, store: IcechunkStore) -> None:
 
     async def test_is_empty(self, store: IcechunkStore) -> None:
         assert await store.is_empty("")
+        await self.set(
```

Comment:
There are no implicit groups in Zarr V3, so I think this is fine. Anyone using the group API would have created these keys.

cc @jhamman

Reply:
`get_node("")` fails because there a root group was not explicitly created, and we go down the "array chunk list" path. I think this is another case where Icechunk is not a key-value store...